schema-registry/rest-service/src/test/resources/schema-registry-client.yaml

```diff
@@ -8,4 +8,11 @@ schema.registry.client.schema.metadata.cache.expiry.interval : 300
 schema.registry.client.schema.text.cache.size : 1024
 schema.registry.client.schema.text.cache.expiry.interval : 300
 schema.registry.client.url.selector : "com.hortonworks.registries.schemaregistry.client.FailoverUrlSelector"
```

Comment:
Should the URL selector come under `schema.registry.client.retry.strategy` config?

Reply:
We can't move this URL selector config under the retry strategy config, as it will break backward compatibility.